package.json

```diff
@@ -31,12 +31,12 @@
     "xmlbuilder": "9.0.1"
   },
   "devDependencies": {
-    "express": "3.x",
-    "jasmine-node": "1.14.5",
+    "express": "^4.16.3",
+    "jasmine-node": "^1.14.5",
     "jscs": "3.0.7",
     "jsdoc": "3.5.5",
-    "jshint": "2.8.x",
-    "nsp": "2.8.0",
+    "jshint": "^2.9.5",
+    "nsp": "^3.2.1",
```

Comment:
@dkundel Looks like this version bump is breaking node v5 + v4. We'll probably try to move to `npm audit` in the future, but do you think rolling this back is fine for now?

Reply:
Yeah I'm fine with it now. Can you log a bug to upgrade them later? 

But v4 + v5 are both deprecated so we should definitely target moving off them soon. I think security is more important in this case.

Reply:
Ah, not just deprecated but [EOL](https://github.com/nodejs/Release)-ed. So is v7. I'll log a ticket to coordinate dropping them.